lib/tasks/foreman_templates_tasks.rake

```diff
@@ -57,6 +57,13 @@ namespace :templates do
     }).purge!
   end
 
+  desc 'Clean data created by this plugin, this will permanently delete the data!'
```

Comment:
Hm, could you find some more generic message? I'd like this to be generic for future. Currently it only deletes Settings but the concept should be also for any default data coming from this plugin.